lib/l10n/app_en.arb

```
@@ -0,0 +1,631 @@
+    "callDeclined": "Call declined",
+    "callDeclinedTo": "User ${username} declined the call.",
+    "@callDeclinedTo": {
+        "description": "Message when a call is declined",
+        "placeholders": {
+            "username": {
+                "type": "String",
+                "example": "john_doe"
+            }
+        }
+    },
+    "checkForUpdates": "Check Updates",
```

RuboCop (1.76.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)